packages/flutter_robusta_firebase_core/lib/src/extension.dart

```diff
@@ -0,0 +1,27 @@
+import 'dart:async';
+
+import 'package:firebase_core/firebase_core.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:robusta_runner/robusta_runner.dart';
+
+/// {@template flutter_robusta_firebase_core}
+/// A Helper class for Firebase core initialization.
+/// {@endtemplate}
+class FlutterFirebaseCoreExtension implements Extension {
+  /// {@macro flutter_robusta_firebase_core}
+  const FlutterFirebaseCoreExtension({String? name, FirebaseOptions? options})
+      : _name = name,
+        _firebaseOptions = options;
+
+  final String? _name;
+  final FirebaseOptions? _firebaseOptions;
+
+  @override
+  void load(Configurator configurator) {
+    configurator.addBoot(_boot, priority: 5429);
```

Comment:
The priority should be `2^n`, n should be lower than 8192 (priority of `FlutterAppExtension`).

Reply:
Could you elaborate more about this. Still haven't got what your idea is.